raiden/network/transport/matrix/transport.py

```diff
@@ -1199,7 +1287,11 @@ def _send_raw(self, receiver_address: Address, data: str) -> None:
                 room=room,
                 data=data.replace("\n", "\\n"),
             )
-            room.send_text(data)
+            if self._web_rtc_manager.has_ready_channel(receiver_address):
+                rtc_partner = self._web_rtc_manager.address_to_rtc_partners[receiver_address]
+                send_message(rtc_partner, data, self._raiden_service.address)
```

Comment:
Nitpick: Including `rtc` or so in the name would make it a bit easier to follow.

Also, shouldn't this be able to be moved up above the `_get_room_for_address()` call?

Reply:
IMO, I would leave it as it is. This is due to the fact that there should always exist a room if there is a web rtc channel. Per definition a rtc channel can only exist if a room exist. If there was some odd behavior, we wouldnt notice it if we ask for the rtc channel but not for the room.